dapps/shop/backend/queues/makeOfferProcessor.js

```diff
@@ -0,0 +1,127 @@
+const queues = require('./queues')
+
+const Web3 = require('web3')
```

Comment:
I'd love for us to completely get rid of Web3 and replace it with ether.js. We don't have to do this in this PR. I'll create a separate github issue to track this and we can discuss there.

Reply:
Yea agree we should switch